src/components/filter/HotelId/NewHotels.js

```diff
@@ -1,29 +1,57 @@
-import React from "react";
-import { useParams } from "react-router-dom";
-import HotelRoute from "./RouterHotel";
-//import HotelItem from "../../HotelItem";
-//import AvailableHotels from "../AvailableHotels";
 import nightIcon from '../../../img/icons/Night.png';
 import account from '../../../img/icons/Vector.png';
 import logo_vector from '../../../img/icons/logo_vector.png'
+//import HotelDetail from '../HotelId/RouterHotel'
+import React, { useState, useEffect } from 'react'
+import { useParams } from 'react-router-dom'
 
-const HotelDetail = () => {
-    const {id} = useParams()
-    const thisHotels = HotelRoute.find(prod => prod.id === id)
+  const FetchHotelsRouter = () => {  
 
+    const params = useParams()
 
-    return (
-        <>
+    const [availableHotel, setAvailableHotel] = useState(null)
+    const [error, setError] = useState(null)
+    const [isLoaded, setIsLoaded] = useState(false)
+  
+    useEffect(() => {
+      const url = new URL(
+        'https://fe-student-api.herokuapp.com/api/hotels/hotelId',
+      )
+      const newUrl = new URL(`${params.id}`, url)
```

Comment:
hotelID shouldn't be a part of path, it is a changeable parameter, `https://fe-student-api.herokuapp.com/api/hotels/${hotelId}` , and if you choose hotelId, so in your Route it also should be /hotel/:hotelId